contracts/directory/Makefile

```
@@ -0,0 +1,5 @@
+wasm:
+	RUSTFLAGS='-C link-arg=-s' cargo build --release --target wasm32-unknown-unknown
+
+generate-schema:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)